tests/test_theme.py

```diff
@@ -0,0 +1,226 @@
+# encoding: utf-8
+
+"""
+Test suite for pptx.theme module
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pytest
+
+from pptx.theme import (
+    Theme,
+    ThemeElements,
+    ColorScheme, 
+    FontScheme,
+    FormatScheme,
+    FontCollection,
+)
+from pptx.dml.color import ColorFormat
+from pptx.dml.fill import FillFormat
+from pptx.dml.line import LineStyle
+from pptx.text.text import TextFont
+
+from .unitutil.cxml import element
+
+
+
+class DescribeTheme:
```

Comment:
What about tests to make sure that Theme attributes are read-only?

Reply:
I don't see a reason to include those tests as making Themes read only wasn't a feature, it was just to simplify the implementation since we don't currently have a use case for updating/changing them.  We may want to add the setters at some point.  I'm also not sure how much help those tests would be.  It's not like we are actually going to add a whole setter function by accident.

Reply:
I thought I saw a comment somewhere about making Theme readonly, but if that's the case then it's fine.

Reply:
Yeah, I noted that it is read-only in the docstring somewhere but that's really only because I didn't take the time to implement the implement the write functionality.